src/store/actions.js

```diff
@@ -137,4 +137,48 @@ export default {
   async getHeight({ state: { sdk } }) {
     return (await sdk.topBlock()).height;
   },
+  async claimTips({ getters: { activeNetwork } }, { url, address }) {
+    return fetchJson.post(`${activeNetwork.backendUrl}/claim/submit`, { url, address });
+  },
+  async cacheInvalidateOracle({ getters: { activeNetwork } }) {
+    return fetchJson.get(`${activeNetwork.backendUrl}/cache/invalidate/oracle`);
+  },
+  async cacheInvalidateTips({ getters: { activeNetwork } }) {
+    return fetchJson.get(`${activeNetwork.backendUrl}/cache/invalidate/tips`);
+  },
+  async donateError({ getters: { activeNetwork } }, { error, description }) {
+    return fetchJson.post(`${activeNetwork.backendUrl}/errorreport`, {
+      ...error,
+      description,
+    });
+  },
+  async sendTipComment({ getters: { activeNetwork } }, [tipId, text, author, signCb, parentId]) {
+    const sendComment = async postParam =>
+      fetchJson.post(`${activeNetwork.backendUrl}/comment/api/`, postParam);
+
+    const responseChallenge = await sendComment({ tipId, text, author });
+    const signedChallenge = await signCb(responseChallenge.challenge);
+    const respondChallenge = {
+      challenge: responseChallenge.challenge,
+      signature: signedChallenge,
+      parentId,
+    };
+    return sendComment(respondChallenge);
+  },
+  async modifyNotification({ getters: { activeNetwork } }, [notifId, status, author, signCb]) {
+    const modifyNotif = async postParam =>
```

Comment:
I think we can name this local variable `backendMethod` instead of naming it the same as the whole method, but short.